### ISSUE TITLE
Question: Legality of those streams

### ISSUE THREAD

**Issue by krok55** [notanewbie/LegalStream#5]
Hello,

I'm very interested in your project, but I would really like to know how the list was built and how the legality of each stream was verified.

Thanks

**Comment by notanewbie** [comment]
They are not legally verified per se; however, all theses URLs were procured from the official websites.

**Comment by notanewbie** [comment]
I did just remove the FNC Preview stream (http://fnctve-i.akamaihd.net/hls/live/200669/fnctve/HD.m3u8), as I'm not sure as to its legality. I've emailed Fox Insider explaining that I got the stream URL from their website (http://video.foxnews.com/v/1241186546001/?#sp=watch-live to be exact) and am waiting to hear back.

**Comment by krok55** [comment]
Thanks.
If you get any positive answer, please post it here.

**Comment by notanewbie** [comment]
I will. I'm still waiting for an answer, but the URL remains in operation.

**Comment by notanewbie** [comment]
Meanwhile, I have some copyright concerns with my Steven Universe 24/7 stream.
https://live.ponies.fr/hls/sug-hd/sug.m3u8
It's hosted by live.ponies.fr, which seems to be the some hispanic dude's MLP fanpage. My concerns with the stream is 1) It's not hosted by Turner and 2) It seems like the guy who hosts the site was previously hosting a MLP 24/7 stream that was taken down. This is all speculation, and if you go to ponies.fr nothing happens, but if you view it using Google Translate (https://translate.google.com/translate?sl=auto&tl=en&js=y&prev=_t&hl=en&ie=UTF-8&u=https%3A%2F%2Fponies.fr&edit-text=&act=url) it tries to stream the URL https://live.ponies.fr/hls/hd/ep.m3u8 and fails. I'm assuming that ep stands for episodes, thus the basis for my assumptions

**Comment by notanewbie** [comment]
A while back I removed the Infowars stream, https://infowarslive-lh.akamaihd.net/i/infowarslivestream_1@353459/master.m3u8. It's hidden behind a paywall if you go to Alex Jones' website.

**Comment by krok55** [comment]
Thanks for updates...
Do you have any positive confirmations on the channels in question?

**Comment by notanewbie** [comment]
Nope, but I'm pretty sure posting Infowars' stream was illegal since it's hidden behind a paywall. (I found it in a playlist that included legal streams like local Florida tourism channels but also paid channels like Comedy Central, TheBlaze and One America News.)

**Comment by skydrome** [comment]
Looks like fox is requiring a logon to prove you have a cable subscription to view it

**Comment by notanewbie** [comment]
That was my reasoning behind the move, @skydrome. If it's hidden behind a paywall, I don't want it here.
The Shepard Smith Reporting stream seems to have Shep's breaking news feed, as I caught it when the NYC bike attack was going on.
Honestly, my point in having the FNC Preview Stream is it lets you see Fox News' headlines and get a grasp of what they're talking about right now.

**Comment by notanewbie** [comment]
I was just able to get the Fox Business stream using the same methods (http://fbngohls-i.akamaihd.net/hls/live/263343/FBNGOHLSv2/master.m3u8?hdnea=st=1517508688~exp=1517519488~acl=/*~id=urn_foxnews_com_sp_site_2__7db815d2b2f46cd52b11ca84a13bc3e4e94cab40c3c92f4ae2bc47dbcdff441e~hmac=34a5e64c22075ecdf0aac603e048c7bc2955f0ee8187d8d831f0ad091072a7b6 and https://fbngohls-i.akamaihd.net/hls/live/263343/FBNGOHLSv2/Fox_Business_v2_VIDEO_1_2564000.m3u8) but again, they're illegal.
